pkg/schedule/schedulers/hot_region.go

```diff
@@ -670,9 +672,9 @@ func (bs *balanceSolver) solve() []*operator.Operator {
 				}
 			}
 			bs.cur.mainPeerStat = mainPeerStat
-			if tooHotNeedSplit(srcStore, mainPeerStat, splitThresholds) && bs.GetStoreConfig().IsEnableRegionBucket() {
+			if bs.tooHotNeedSplit(srcStore, mainPeerStat, splitThresholds) && bs.GetStoreConfig().IsEnableRegionBucket() {
```

Comment:
```suggestion
			if bs.GetStoreConfig().IsEnableRegionBucket() &&bs.tooHotNeedSplit(srcStore, mainPeerStat, splitThresholds){
```
how about firstly judge config?